test/suites/standard1/networking-smoke.robot

```diff
@@ -77,6 +74,11 @@ Wait For Service Deletion With Timeout
     Wait Until Keyword Succeeds    30s    1s
     ...    Network APIs With Test Label Are Gone
 
+Restart Router
+    [Documentation]    Testing
```

Comment:
Docs should be updated.

Reply:
Can we add an explanation in a comment (one that cannot be missed easily) on the problem and solution?

Reply:
Done.